app/controllers/start_group_controller.rb

```diff
@@ -1,49 +1,18 @@
 class StartGroupController < ApplicationController
+
   def new
     @errors = []
     @group = Group.new
     if !current_user_or_visitor.is_logged_in?
       render :new
-    elsif current_user_or_visitor.angular_ui_enabled?
+    else current_user_or_visitor.angular_ui_enabled?
```

Comment:
Updating to be a simple if/else; we shouldn't care about angular_ui_enabled? anymore.